src/what-the-graphlet.h

```diff
@@ -0,0 +1,32 @@
+#ifndef WHAT_THE_GRAPHLET_H
+#define WHAT_THE_GRAPHLET_H
+
+#include "blant.h"
+
+
+#define MAXK 9
+
+#if MAXK == 9
+#define MAX_SIZE 3160576
```

Comment:
These are all already defined in blant-fundamentals.h. It's there for a reason. Don't duplicate it.

Reply:
The reason is I implemented what-the-graphlet.c without using anything inside of BLANT